frontend/lib/ui/InputField/InputField.tsx

```diff
@@ -69,7 +71,7 @@ export function InputField({
         </span>
         {disabled ? <div className={`${fieldSize} disabled_input`}>{value}</div> : inputEl}
       </label>
-      {(error || hint) && (
+      {!hideErrorMessage && (error || hint) && (
```

Comment:
I would prefer `showErrorMessage` instead of this double negation (easier to reason about)